RevenueCatUI/CustomerCenter/Data/CustomerCenterActionPreferenceConnector.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterActionPreferenceConnector.swift
+//  
+//  Created by Cesar de la Vega on 2024-06-17.
+
+import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+/// A view modifier that connects CustomerCenterViewModel actions to the SwiftUI preference system
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct CustomerCenterActionPreferenceConnector: ViewModifier {
```

Comment:
I wonder if renaming this to `CustomerCenterActionPreferencesViewModifier` (or something similar) might make things a bit clearer. 

Sometimes these custom suffixes add clarity, but other times they just add noise when navigating the code.